paperless/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: paperless_webserver_1
+      APP_PORT: 8000
+
+  broker:
+    image: docker.io/library/redis:7
+    restart: unless-stopped
+    volumes:
+      - ${APP_DATA_DIR}/redisdata:/data
+
+  webserver:
+    image: ghcr.io/paperless-ngx/paperless-ngx:latest
```

Comment:
`image: ghcr.io/paperless-ngx/paperless-ngx:2.7.2@sha256:703c990a790dfd4d25fb56df3afec27b13cb0926a3818bf265edac9c71311647`

Reply:
Yeah, sorry, I forgot to pin the version, thanks for pointing that out. Fixed it!